test/ibmq/test_ibmq_client_v2.py

```diff
@@ -149,26 +148,56 @@ def test_api_version(self, qe_token, qe_url):
         version = api.api_version()
         self.assertIsNotNone(version)
 
-    @skip('TODO: reenable after checking with API team')
     @requires_qe_access
     @requires_new_api_auth
     def test_get_job_includes(self, qe_token, qe_url):
-        """Check the field includes parameter for get_job."""
+        """Check the include fields parameter for get_job."""
         IBMQ.enable_account(qe_token, qe_url)
 
-        backend_name = 'ibmq_qasm_simulator'
-        backend = IBMQ.get_backend(backend_name)
-        circuit = transpile(self.qc1, backend, seed_transpiler=self.seed)
-        qobj = assemble(circuit, backend, shots=1)
+        api, job = self._submit_job_to_backend('ibmq_qasm_simulator')
+        job_id = job['id']
 
-        api = backend._api
-        job = api.submit_job(qobj.to_dict(), backend_name)
+        # Get the job, including a field.
+        self.assertIn('backend', job)
+        self.assertIn('shots', job)
+        job_included = api.get_job(job_id, include_fields=['backend'])
+
+        # Ensure the result has only the included field
+        self.assertIn('backend', job_included)
+        self.assertNotIn('shots', job_included)
+
+    @requires_qe_access
+    @requires_new_api_auth
+    def test_get_job_excludes(self, qe_token, qe_url):
+        """Check the exclude fields parameter for get_job."""
+        IBMQ.enable_account(qe_token, qe_url)
+
+        api, job = self._submit_job_to_backend('ibmq_qasm_simulator')
         job_id = job['id']
 
-        # Get the job, excluding a parameter.
+        # Get the job, excluding a field.
+        self.assertIn('shots', job)
+        self.assertIn('backend', job)
+        job_excluded = api.get_job(job_id, exclude_fields=['backend'])
+        # Ensure the result only excludes the specified field
+        self.assertNotIn('backend', job_excluded)
         self.assertIn('shots', job)
-        job_excluded = api.get_job(job_id, exclude_fields=['shots'])
-        self.assertNotIn('shots', job_excluded)
+
+    @requires_qe_access
+    @requires_new_api_auth
+    def test_get_job_with_invalid_filter(self, qe_token, qe_url):
```

Comment:
Can you revise this test's name and docstring? "Invalid" might be a bit ambiguous, as the filter should end up being well-formed but containing non-existent fields.